src/cloud/audiocom/audiocomservice.cpp

```diff
@@ -372,6 +372,11 @@ Ret AudioComService::doCreateAudio(network::INetworkManagerPtr manager, const QS
     return ret;
 }
 
+QUrl AudioComService::audioComUrl() const
```

Comment:
You can get this info through AudioComService::cloudInfo().url, so don't need to create a new method